conda_smithy/configure_feedstock.py

```diff
@@ -89,7 +89,7 @@ def compute_build_matrix(meta):
 
 def main(forge_file_directory):
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'pelson/obvious-ci:latest_x64', 'command': 'bash'},
+    config = {'docker': {'image': 'jjhelmus/conda-forge-x86_64:latest', 'command': 'bash'},
```

Comment:
I'd be more comfortable if this were under a conda-forge organisation on dockerhub. Does such a concept exist? Would you mind setting it up and adding yourself, me, @msarahan and @ocefpaf as administrators?